packages/zorb-contracts/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "author": "iain nash",
-  "license": "GPLv3",
+  "license": "GPL-3.0-only",
```

Comment:
This change is informed by the official SPDX guidelines for this license:

https://spdx.org/licenses/GPL-3.0-only.html